docs/config.md

```diff
@@ -235,3 +235,9 @@ Instead of tagging, you also have the option to add metadata to instances. This
         securityGroups:
    ...
    ```
+
+## Timeout settings
+During some heavy workload cloud, the time for create and delete openstack instance might takes long time, by default it's 5 minute.
+you can set:
+``CLUSTER_API_OPENSTACK_DELETE_TIMEOUT`` for instance delete timeout value.
+``CLUSTER_API_OPENSTACK_CREATE_TIMEOUT`` for instance create timeout value.
```

Comment:
How about 

- s/CLUSTER_API_OPENSTACK_DELETE_TIMEOUT/CLUSTER_API_OPENSTACK_INSTANCE_DELETE_TIMEOUT
- s/CLUSTER_API_OPENSTACK_CREATE_TIMEOUT/CLUSTER_API_OPENSTACK_INSTANCE_CREATE_TIMEOUT

You only need one `

Reply:
ok